apiserver/alembic/versions/201710031816_7f0054256cf5_initial_database_schema.py

```diff
@@ -360,7 +360,7 @@ def upgrade():
         'user_tier_history',
         sa.Column('user_id', mysql.MEDIUMINT(display_width=8, unsigned=True),
                   autoincrement=False, nullable=False),
-        sa.Column('tier', mysql.VARCHAR(length=256), nullable=False),
+        sa.Column('tier', mysql.VARCHAR(length=127), nullable=False),
```

Comment:
This file defines base reality for the start of alembic revisions and in general shouldn't be modified unless the definitions actually don't match reality.

Having said that I think all the changes here were a mis-commit as the commit message was "Add mu/sigma/rank fields to game_participant", but only changes to this file were made.

Reply:
Ah yeah, this is a mis-commit. (I made a local copy of the database, but the MySQL version I used didn't like the length of some of the fields.)